packages/next/client/dev/error-overlay/hot-dev-client.js

```diff
@@ -325,16 +369,12 @@ async function tryApplyUpdates(onHotUpdateSuccess) {
   }
 
   // https://webpack.github.io/docs/hot-module-replacement.html#check
-  try {
-    const updatedModules = await module.hot.check(
-      /* autoApply */ {
-        ignoreUnaccepted: true,
-      }
-    )
-    if (updatedModules) {
+  module.hot.check(/* autoApply */ true).then(
```

Comment:
Should we update the above link with latest `https://webpack.js.org/api/hot-module-replacement/#check` current one redirects to home page of `webpack.js.org` after 3 seconds